protolib2/traverse.py

```diff
@@ -0,0 +1,179 @@
+import glob
+# import logging
+import os
+import os.path
+
+from protolib import categories
+from protolib2.parse import (
+    parseOT1 as OT1_parser,
```

Comment:
These 2 cannot coexist in the same script because each needs a different version of `opentrons` library. That's why the virtualenvs are needed